src/test/java/org/springframework/data/tarantool/repository/BookRepositoryWithTuple.java

```diff
@@ -0,0 +1,15 @@
+package org.springframework.data.tarantool.repository;
+
+import org.springframework.data.tarantool.core.mapping.Tuple;
+import org.springframework.data.tarantool.entities.Book;
+
+
+/**
+ * @author Oleg Kuznetsov
+ */
+@Tuple("test_space")
+public interface BookRepositoryWithTuple extends TarantoolRepository<Book, Integer> {
```

Comment:
Class BookRepository has a @Tuple annotation but without parameters. Maybe rename them  to BookRepositoryWithTupleWithSpaceParam and BookRepositoryWithTupleWithDefaults ? 
Also then we need testcases for a repository without @Tuple..